overlord/snapstate/snapmgr.go

```diff
@@ -556,6 +556,45 @@ func (m *SnapManager) RefreshSchedule() (string, bool, error) {
 	return m.autoRefresh.RefreshSchedule()
 }
 
+// EnsureAutoRefreshesAreDelayed will delay refreshes for the specified amount
+// of time, as well as return any active auto-refresh changes that are currently
+// ready so that the client can wait for those.
```

Comment:
I guess `not ready`, right?

Reply:
sorry what do you mean? The changes returned are "Ready" as in they are either running or about to be run, and as such can't be delayed (easily), and so the client ought to wait for them.

Reply:
I think you meant to write this:
```
 // EnsureAutoRefreshesAreDelayed will delay refreshes for the specified amount
// of time, as well as return any active auto-refresh changes that are currently
// **not** ready so that the client can wait for those.
```

Reply:
ah sorry I see what you mean now, yes you are right